core/src/SYCL/Kokkos_SYCL_Parallel_Scan.hpp

```diff
@@ -86,9 +86,9 @@ class ParallelScanSYCLBase {
   void scan_internal(sycl::queue& q, const Functor& functor,
                      pointer_type global_mem, std::size_t size) const {
     // FIXME_SYCL optimize
-    constexpr size_t wgroup_size = 32;
+    constexpr size_t wgroup_size = 128;
```

Comment:
This change is not really necessary of course but is more in line with what we are doing for reductions at the moment.